makelib/uptest.mk

```diff
@@ -0,0 +1,66 @@
+# ====================================================================================
+# End to End Testing
+
+UPTEST_EXTRA_ARGS ?=
+
+UPTEST_COMMAND = SKIP_DEPLOY_ARGO=$(SKIP_DEPLOY_ARGO) \
+	KUBECTL=$(KUBECTL) \
+	CHAINSAW=$(CHAINSAW) \
+	CROSSPLANE_CLI=$(CROSSPLANE_CLI) \
+	CROSSPLANE_NAMESPACE=$(CROSSPLANE_NAMESPACE) \
+	YQ=$(YQ) \
+	$(UPTEST) e2e $(UPTEST_CLAIMS) \
```

Comment:
Here we have some configuration specific naming `$(UPTEST_CLAIMS)`. We also use the uptest runs in provider repositories and in these repos our test inputs are Managed Resources not Claims.

So my suggestion is having a more generic name for `UPTEST_CLAIMS` like `UPTEST_MANIFESTS` (not sure naming), for pointing this is for configuration testing. Otherwise, this may need a confusion for users who want to use this reusable targets and variables.